__tests__/app.test.js

```diff
@@ -156,4 +156,26 @@ describe("app ", () => {
         });
     });
   });
+  describe("POST /api/reviews/:review_id/comments ", () => {
+    test("respond with 201 when a valid comment is posted", () => {
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .send({ username: "philippaclaire9", body: "test comment" })
+        .expect(201);
+    });
+    test("return status 201, responds with an array of comments for the given review_id ", () => {
```

Comment:
This isn't responding with an array of comments, rather the comment that was posted, ✉️  so I would edit this test's description to  reflect that 🙂